federation/cmd/federation-controller-manager/app/options/options.go

```diff
@@ -35,6 +35,10 @@ type ControllerManagerConfiguration struct {
 	Port int `json:"port"`
 	// address is the IP address to serve on (set to 0.0.0.0 for all interfaces).
 	Address string `json:"address"`
+	// federation name.
+	FederationName string `json:"federationName"`
```

Comment:
Can I choose any name? Does it have to match a DNS config parameter?
Or any other parameter anywhere else?